### PR TITLE
Fixed valid meshconfig error

### LINKED ISSUE #5905: [mesheryctl] Failure to recognize valid meshconfig
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
```
$ mesheryctl system channel switch stable
Valid context is not available in meshconfig
Error: failed to initialize healthchecker: Valid context is not available in meshconfig
```
```
$ mesheryctl system context view
Error: current context not set
```
```
$ mesheryctl system start
Valid context is not available in meshconfig
Error: Valid context is not available in meshconfig
```
```
$ cat ~/.meshery/config.yaml
contexts:
    local:
        endpoint: http://localhost:9081
        token: Default
        platform: kubernetes
        components:
            - meshery-app-mesh
            - meshery-istio
            - meshery-linkerd
            - meshery-consul
            - meshery-nsm
            - meshery-kuma
            - meshery-osm
            - meshery-traefik-mesh
            - meshery-nginx-sm
            - meshery-cilium
        channel: edge
        version: latest
current-context: local
tokens:
    - location: auth.json
      name: Default
```

#### Environment

- Host OS: Mac 
- Meshery Server Version: stable-v0.6.0-rc.6d
- Meshery Client Version: stable-v0.6.0-rc.6d
- Platform: Kubernetes

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by harkiratsm**
Signed-off-by: harkiratsm <multaniharry714@gmail.com>

**Description**

This PR fixes #5905 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by alphaX86** [comment]
Testing the PR now...

File: mesheryctl/internal/cli/root/root.go [hunk 1]
```diff
@@ -170,16 +170,17 @@ func initConfig() {
...
 
-		// If a config file is found, read it in.
-		if err := viper.ReadInConfig(); err == nil {
-			log.Debug("Using config file:", viper.ConfigFileUsed())
-		}
+	// If a config file is found, read it in.
+	if err := viper.ReadInConfig(); err == nil {
+		log.Debug("Using config file:", viper.ConfigFileUsed())
 	}
+
```
Comment (warunicorn19) [inline mesheryctl/internal/cli/root/root.go:183]:
```suggestion
```

**Review by alphaX86 — COMMENTED** [review]
So it's the position and spacing that caused errors huh... So far it works good, testing system commands now...

**Review by alphaX86 — APPROVED** [review]
All commands worked fine now. LGTM great job @harkiratsm

**Review by nishantwrp — APPROVED** [review]
lgtm, thanks for the quick fix @harkiratsm

**Review by warunicorn19 — APPROVED** [review]
Thank you @harkiratsm

**Pull request merged**